doc/pedersen_commitments/pedersen-commitments.md

```diff
@@ -0,0 +1,109 @@
+# Pedersen Commitments
+
+In order to obfuscate the amount of energy that is being transferred, the amount of energy is encrypted using Pedersen commitments. The Pedersen commitment is a commitment scheme that allows the holder of the commitment to commit to a value without revealing the value itself. The commitment can be opened to reveal the value. The holder cannot open it to reveal another value. The commitment scheme is homomorphic in the sense that the product of two commitments will be a commitment to the sum of the two values.
+
+## Pedersen commitments are defined as follows:
+$$c=C(m,r)$$
+|identifier|property|usage|accessability|
+|---|---|---|---|
+`c` | commitment | the commitment itself | public |
+`m` | message | the message that is being committed to | private |
+`r` | randomness | the randomness used to commit to the message | private |
+
+```mermaid
+graph TD
+    A[Commitment] --> B[Message]
+    A --> C[Randomness]
+```
+
+### Additional properties of Pedersen commitments
+There are commutation and homomorphism properties that are useful for the Pedersen commitment scheme. These properties are defined as follows:
+
+
+## What these properties can do in practice for Energy Track and Trace
+The Pedersen commitment scheme is used to obfuscate the amount of energy that is being transferred. The amount of energy is encrypted using Pedersen commitments. An additional property is that we can add and subtract hidden amounts of energy without revealing the individual amounts to anyone - but you can prove that the result is correct.
+
+## How to prove that a certain value is committed to without revealing the value itself
+```mermaid
+flowchart LR
+
+    subgraph S1[Value]
+        direction TB
+        A[Wh] 
+        subgraph S2[Pedersen Commitments]
+            direction TB
+            B[C_total]
+            
+            subgraph S3[Commited Transactions]
+                C[C_t1]
+                D[C_t2]
+                E[C_tN]
+                F[C_remainder]
+            end
+            
+            
+            subgraph S4[Validation]
+                direction LR
+                J[C_total / Commited Transacations]
+                M[Commitment to 0]
+            end
+
+            subgraph S5[Proof]
+                direction LR
+                K[m_t1]
+                L[r_t1]
+            end
+        end
+    end
+
+    A --> B
+    B --> |/| C
+    C --> |*| D
+    D --> |*| E
+    E --> |*| F
+    J --> |=| M
+    S5 --> C
+```
+
+### Pedersen commitments are used in the following way to obfuscate the amount of energy that is being transferred:
+We want to show that the sum of the commitments in a certificate (e.g. transactions + remainder) is equal to the committed value in the original production certificate, 
+corresponding to 
+
+$$Ctotal / (Ctransactions * Cremainder)$$ 
```

Comment:
C´ or Commitment to 0 == ...

Maybe it should be more explicit that it is not normal devide and times, but simple the notation one uses.